lib/storage/file.js

```diff
@@ -325,8 +325,10 @@ File.prototype.createReadStream = function(options) {
     };
 
     if (rangeRequest) {
+      var start = typeof options.start === 'undefined' ? '' : options.start;
+      var end = typeof options.end === 'undefined' ? '' : options.end;
```

Comment:
I considered that. But they could technically pass a string and it would still work as long as it was a number e.g. `"3"` I can switch to using `util.is()` if you prefer that though.